### PR TITLE
framework: Add Intel Core Ultra Series 1

### PR DESCRIPTION
**Pull request by JohnAZoidberg**
###### Description of changes

Not selling to customers yet, but already announced: https://frame.work/tw/en/blog/introducing-the-new-framework-laptop-13-with-intel-core-ultra-series-1-processors

###### Things done

- [x] Tested the changes in your own NixOS Configuration
- [x] Tested the changes end-to-end by using your fork of `nixos-hardware` and
      importing it via `<nixos-hardware>` or Flake input

### CODE REVIEW

**Comment by quinn-dougherty** [comment]
maybe shorter name? like `ultra-1`. Even better (maybe), `1-ultra` to fit with `<number>-gen` pattern from previous intel regime.

File: README.md [hunk 1]
```diff
@@ -150,6 +150,7 @@ See code for all available configurations.
 | [Framework 11th Gen Intel Core](framework/13-inch/11th-gen-intel)      | `<nixos-hardware/framework/13-inch/11th-gen-intel>`     |
 | [Framework 12th Gen Intel Core](framework/13-inch/12th-gen-intel)      | `<nixos-hardware/framework/13-inch/12th-gen-intel>`     |
 | [Framework 13th Gen Intel Core](framework/13-inch/13th-gen-intel)      | `<nixos-hardware/framework/13-inch/13th-gen-intel>`     |
+| [Framework Intel Core Ultra Series 1](framework/13-inch/intel-core-ultra-series1) | `<nixos-hardware/framework/13-inch/intel-core-ultra-series1>`     |
```
Comment (Mic92) [inline README.md:153]: Please also reference this in the flake.nix.

**Comment by umgefahren** [comment]
I would love to see this shipped :)

**Comment by umgefahren** [comment]
For anyone interested: I addressed the concerns raised above and submitted a PR to Framework. Here it is: https://github.com/FrameworkComputer/nixos-hardware/pull/1
I have been running this patch ever since and I have no complaints. I would be very happy to see @FrameworkComputer merge it.
I could also just open a new PR to this Repo.

**Comment by Mic92** [comment]
@umgefahren Just open a new PR.

**Comment by dbirks** [comment]
Just opened a new PR #1250 and pulled in everyone's changes.

**Pull request closed without merging**